cef/CustomEditField/CustomEditField.rbbas

```diff
@@ -1190,11 +1190,19 @@ Implements MessageReceiver
 		  if SelLength = 0 then Return
 		  dim c as new Clipboard
 		  
+		  Dim textToCopy as String
+		  
 		  #if EditFieldGlobals.Replace00With01
-		    c.Text = me.SelText.ReplaceAll (Chr(1), Chr(0))
+		    textToCopy = me.SelText.ReplaceAll (Chr(1), Chr(0))
 		  #else
-		    c.Text = me.SelText
+		    textToCopy = me.SelText
+		  #endif
+		  
+		  #if TargetWindows
```

Comment:
This does not compile in RB 2012, where only "TargetWin32" is defined, so please wrap this check into "#if RBVersion >= 2013"